exercises/series/.meta/src/reference/java/Series.java

```diff
@@ -7,30 +7,35 @@
 class Series {
 
     private final int digitsSize;
-    private List<Integer> digits;
+    private List<String> digits;
 
     Series(String string) {
-        this.digits = Arrays.stream(string.split(""))
-                        .map(Integer::parseInt)
-                        .collect(Collectors.toList());
-        this.digitsSize = this.digits.size();
+        this.digits = Arrays.stream(string.split("")).collect(Collectors.toList());
+        if(!string.isEmpty()) {
+            this.digitsSize = this.digits.size();
+        }else {
+            this.digitsSize = 0;
+        }
     }
 
-    List<List<Integer>> slices(int num) {
+    List<List<String>> slices(int num) {
```

Comment:
Sorry, I probably wasn't quite clear about what I was suggesting this method should return. Instead of returning a list of lists of strings, where each string represents a digit, you could return a list of strings where each string is a number. That way it would match the canonical data 🙂 Does that make sense? 

If you don't want to do that as part of this review, that's fine. I can open another issue for it 🙂

Reply:
@FridaTveit , I got the request. And I want to fix that but, I would like to do it as separate issue.

Reply:
Please let me know once you open an issue for that, I'll raise a new PR.

Reply:
I've openened issue #1598 🙂